apps/sensenet/src/components/drawer/PermanentDrawer.tsx

```diff
@@ -108,7 +109,12 @@ const PermanentDrawer: React.FunctionComponent<RouteComponentProps> = props => {
             ) : null}
```

Comment:
Can you also please get rid of the warning `Material-UI: the 'anchorEl' prop provided to the component is invalid.` by splitting the open and close button to have a separate Tooltip parent?

Reply:
This solution won't solve this warning :(